### PR TITLE
270 sign up improvements

### PR DESCRIPTION
**Pull request by iandjx**
Updated login and register pages based on Figma spec
Use [React Email](https://react.email/) for email templating of account validation, invite, and password reset emails

### CODE REVIEW

File: webapp/src/emails/Verification.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+								<Img
+									src='https://mintlify.s3-us-west-1.amazonaws.com/rna/logo/light.svg'
+									height='50'
+									alt="AgentCloud's Logo"
+								/>
+							</Section>
+							<Section className='p-6'>
+								<Heading className='text-gray-800 font-bold text-xl mb-4'>Welcome to Agent Cloud</Heading>
+								<Text className='text-gray-800 mb-4'>
+                                    You’re ine step away from completing  your account registration. Please confirm your account through the link
```
Comment (tomlynchRNA) [inline webapp/src/emails/Verification.tsx:54]: ine -> one

File: webapp/src/emails/Verification.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+								</Text>
+								<Text className='text-gray-800 mb-4'>
+                                    You’re ine step away from completing  your account registration. Please confirm your account through the link
```
Comment (tomlynchRNA) [inline webapp/src/emails/Verification.tsx:57]: This line is repeated

File: webapp/src/pages/register.tsx [hunk 3]
```diff
@@ -47,132 +55,141 @@ export default function Register() {
...
-											: <EyeSlashIcon className='h-5 w-5 text-gray-400' aria-hidden='true' />}
-									</div>
+							<div className='relative mt-2'>
+								<InputField<RegisterFormValues>
+									name='password'
+									control={control}
+									rules={{
+										required: 'Password is required',
+										pattern: {
+											value: /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/,
```
Comment (tomlynchRNA) [inline webapp/src/pages/register.tsx:117]: This regex rejects a valid password such as `y4s0BhuWulTAVqQp+4C5tPpvgvU=`

File: webapp/src/emails/Verification.tsx (hunk 1 above)
Comment (iandjx) [reply webapp/src/emails/Verification.tsx]: resolved

File: webapp/src/emails/Verification.tsx (hunk 2 above)
Comment (iandjx) [reply webapp/src/emails/Verification.tsx]: duplicate removed

**Pull request merged**